Update What's New page (#1669)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

<!-- BEGIN: What's New Entries -->

## June 14-18, 2021

- Released grommet-icons v4.6.0! Check out the [Release Notes](https://github.com/grommet/grommet-icons/releases/tag/v4.6.0).

> - Added [download links for MetricHPEXS](/foundation/typography#fonts-for-offline-usage) for offline usage

Reviewer: Opinion: for consistency and since our bullets read as sentences, recommend concluding each bullet point with a period.

```suggestion
- Added [download links for MetricHPEXS](/foundation/typography#fonts-for-offline-usage) for offline usage.
```

Reviewer: ```suggestion
- Added [download links for MetricHPEXS](/foundation/typography#fonts-for-offline-usage) font files for offline usage.
```

> - Enhanced filtering examples to demonstrate [Filtering with selectable results](/templates/filtering#filtering-with-selectable-results)

Reviewer: ```suggestion
- Enhanced filtering examples to demonstrate [Filtering with a table containing selectable rows](/templates/filtering#filtering-with-selectable-results).
```

> - Added guidance around [setting the width of a table](/components/table#setting-the-width-of-the-table)

Reviewer: ```suggestion
- Added guidance for [setting the width of a table](/components/table#setting-the-width-of-the-table).
```

## June 7-11, 2021

- Released Grommet v2.17.3! Read the [Release Notes](https://github.com/grommet/grommet/releases/tag/v2.17.3).
- [Updated Wizard "Next" button mobile layout](/templates/wizard) to match desktop layout

> - Added examples for how Ordering of Items should be handled in a [Table](/templates/table-customization) and [List](/templates/lists#item-order)

Reviewer: ```suggestion
- Added examples for how the **order of items** may be customized in a [Table](/templates/table-customization) and [List](/templates/lists#item-order).
```

> - Updated guidance for [implementing the latest HPE Global Header](/templates/global-header)

Reviewer: ```suggestion
- Provided instructions and guidance for [implementing the latest HPE Global Header](/templates/global-header).
```

## May 31 - June 4, 2021

- Added guiance and examples for [FileInput](/components/fileinput)
- Added `kind="toolbar"` to button in grommet-theme-hpe for use in Filter contexts. See how this button is used in [Filtering examples](/templates/filtering).

## May 24-28, 2021

> - Began a Getting Started with HPE Design System starter app to provide a step-by-step introduction to Grommet, grommet-theme-hpe, and developing with the Design System. Check out the repository: https://github.com/grommet/hpe-design-system-starter

Reviewer: ```suggestion
- Began a "Getting Started with HPE Design System" starter application providing a step-by-step introduction to Grommet, grommet-theme-hpe, and developing with the Design System. Check out the repository: https://github.com/grommet/hpe-design-system-starter.
```

## May 17-21, 2021

- Added guidance for [Spinner](/components/spinner)

> - Improved Select accessibility by allowing caller to set an a11yTitle: https://github.com/grommet/grommet/pull/5234

Reviewer: ```suggestion
- Improved accessibility for Select by [allowing callers to specify an `a11yTitle`](https://github.com/grommet/grommet/pull/5234).
```

> - [Added SkipLinks to the Design System website](https://github.com/grommet/hpe-design-system/pull/1606) to improve experience for keyboard navigators and screen reader users

Reviewer: ```suggestion
- Improved the Design System site experience for keyboard navigators and screen reader users by [adding SkipLinks](https://github.com/grommet/hpe-design-system/pull/1606). 
```

- Completed design explorations for [Ordering of Items](https://www.figma.com/file/TonE8uQ33e5QJRPknZx36q/Ordering-Of-Items?node-id=343%3A6850)

## May 10-14, 2021

- Released grommet-theme-hpe v2.1.0! Read the [Release Notes](https://github.com/grommet/grommet-theme-hpe/releases/tag/v2.1.0).

## May 3-7, 2021

- Improved accessibility of grouped form inputs in Grommet: https://github.com/grommet/grommet/pull/5209

## April 26-30, 2021

- Released Grommet v2.17.2! Read the [Release Notes](https://github.com/grommet/grommet/releases/tag/v2.17.2).

> - Provided example of [CheckBox without a border](https://github.com/grommet/hpe-design-system/issues/1592)

Reviewer: ```suggestion
- Provided example of [CheckBox validation when not contained in a FormField](https://github.com/grommet/hpe-design-system/issues/1592).
```

> - Enhanced grommet-theme-hpe z-index for Layer and Drop to work with HPE Global Header: https://github.com/grommet/grommet-theme-hpe/pull/185

Reviewer: ```suggestion
- Modified grommet-theme-hpe [z-index for Layer and Drop](https://github.com/grommet/grommet-theme-hpe/pull/185) to work with HPE Global Header-Footer Service.
```

## April 19-23, 2021

> - Added [examples of text sizes 3xl, 4xl, 5xl, 6xl](/foundation/typography#text)

Reviewer: ```suggestion
- Added [examples for Text sizes 3xl, 4xl, 5xl, 6xl](/foundation/typography#text).
```

- Updated [Form examples](/templates/forms#shipping): when referring to ZIP codes, the acronym should always be written as ZIP instead of Zip

> - Added guidance around [CheckBox as a toggle](https://design-system.hpe.design/components/checkbox#toggle)

Reviewer: ```suggestion
- Added guidance for [CheckBox as a toggle](https://design-system.hpe.design/components/checkbox#toggle).
```

> - Updated Figma to provide a [RadioButtonGroup with no description](https://www.figma.com/file/aZyY606ENQedz4FXugdzgS/HPE-Radio-Button-Group-Component?node-id=105%3A643)

Reviewer: ```suggestion
- Updated Figma to provide an example for a [RadioButtonGroup without a FormField description](https://www.figma.com/file/aZyY606ENQedz4FXugdzgS/HPE-Radio-Button-Group-Component?node-id=105%3A643).
```

## April 12-16, 2021

- Added guidance and examples for [Tip](/components/tip)

## April 5-9, 2021

- Updated [Wizard guidance](/templates/wizard) and patterns to demonstrate recommended behavior for width of the inputs and footer
- Demonstrated [how to use `plain: true` in DataTable columns](/components/table#selecting-multiple-records-and-batch-actions) to create clickable cells

## March 29 - April 2, 2021

> - Updated guidance around [user-entered input in Forms](/templates/forms#user-entered-input)

Reviewer: ```suggestion
- Updated guidance for [user-entered input in Forms](/templates/forms#user-entered-input).
```

> - Updated guidance around [considerations to make dashboards easier to consume](/templates/dashboards#guidance)

Reviewer: ```suggestion
- Updated guidance for [considerations to make dashboards easier to consume](/templates/dashboards#guidance).
```

> - Added guidance and example around [navigation via Table](/components/table#navigation-via-table)

Reviewer: ```suggestion
- Added guidance and an example for [navigation from within a Table](/components/table#navigation-via-table).
```
